src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -1,32 +1,42 @@
 import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
 import { TemplateResult } from "lit-html";
 import "@polymer/paper-checkbox/paper-checkbox";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
 
+import processConfigEntities from "../common/process-config-entities";
 import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
 import { HomeAssistant } from "../../../types";
 import { LovelaceCardEditor } from "../types";
 import { fireEvent } from "../../../common/dom/fire_event";
-import { Config } from "../cards/hui-glance-card";
+import { Config, EntityConfig } from "../cards/hui-glance-card";
 
 import "../../../components/entity/state-badge";
 import "../../../components/entity/ha-entity-picker";
+import "../components/hui-theme-select-editor";
 import "../../../components/ha-card";
 import "../../../components/ha-icon";
 
 export class HuiGlanceCardEditor extends hassLocalizeLitMixin(LitElement)
   implements LovelaceCardEditor {
   public hass?: HomeAssistant;
   private _config?: Config;
+  private _configEntities?: EntityConfig[];
 
   static get properties(): PropertyDeclarations {
     return {
       hass: {},
       _config: {},
+      _configEntities: {},
     };
   }
 
   public setConfig(config: Config): void {
     this._config = { type: "glance", ...config };
+    const entities = processConfigEntities(config.entities);
```

Comment:
`processConfigEntities` might not be the right function here, as we should be able to edit configs in progress, which might not be valid. `processConfigEntities` will raise if it's invalid.

Reply:
I can create my own function as a helper for this then

Reply:
yes